cmd/containers-storage/layer.go

```diff
@@ -35,6 +35,37 @@ func layer(flags *mflag.FlagSet, action string, m storage.Store, args []string)
 	return 0
 }
 
+func layerParentOwners(flags *mflag.FlagSet, action string, m storage.Store, args []string) int {
+	matched := []*storage.Layer{}
+	for _, arg := range args {
+		if layer, err := m.Layer(arg); err == nil {
+			matched = append(matched, layer)
+		}
+	}
+	if jsonOutput {
+		json.NewEncoder(os.Stdout).Encode(matched)
```

Comment:
The JSON output is a list of (details of?) layers, but that seems to be different from the `LayerParentOwners` computation in the non-JSON case.